src/main/java/com/mg105/entities/Inventory.java

```diff
@@ -0,0 +1,61 @@
+package com.mg105.entities;
+import java.util.ArrayList;
+
+public class Inventory {
+    final static int inventoryLimit = 10;
+    private static ArrayList<Item> items = new ArrayList<>();
+
+    public static int getInventoryLimit(){
+        return inventoryLimit;
+    }
+
+    public static int getNumberOfItems(){
+        return items.size();
+    }
+
+    public static boolean isInventoryFull(){
+        return getNumberOfItems() >= getInventoryLimit();
+    }
+
+    /**
+     * Precondition: Inventory is not full
+     * If precondition is violated, the function throws a RuntimeException
+     * */
```

Comment:
This isn't standard java doc... but do we even want to do standard java doc? (something to discuss next Monday)